### PR TITLE
Add docker ps for old Docker versions

### PR DESCRIPTION
**Pull request by gbarr01**
- Remove prompt signs
- Add numbering
- Tidy text

### CODE REVIEW

File: get-started/index.md [hunk 1]
```diff
@@ -175,19 +177,22 @@ This message shows that your installation appears to be working correctly.
...
 
 ```shell
-$ docker container ls --all
+docker container ls --all
+
 CONTAINER ID     IMAGE           COMMAND      CREATED            STATUS
 54f4984ed6a8     hello-world     "/hello"     20 seconds ago     Exited (0) 19 seconds ago
 ```
 
+> If you have docker-ce 1.12 or lower, run `docker ps --all`. For commands introduced in docker-ce 1.13, see [Introducing Docker 1.13](https://blog.docker.com/2017/01/whats-new-in-docker-1-13/){: target="_blank" class="_"}.
```
Comment (joaofnfernandes) [inline get-started/index.md]: I'm OK with this note, but I think a good assumption to make is that users going through the getting started are just starting with Docker. That means we can assume they have the latest stable version of Docker, and we don't have to put in a bunch of notes explaining things that are relevant for users running an older version.

Comment (gbarr01) [reply get-started/index.md]: That was my thinking ... I have an idea.

**Comment by gbarr01** [comment]
Actually, @joaofnfernandes , I agree. Version 1.12 is far too old and we make it clear that a supported version should be used. I have other small changes however that I would like to incorporate.

**Comment by joaofnfernandes** [comment]
I'm 👍 on all the other changes here.

**Pull request merged**